packages/web/components/earn/table/cells.tsx

```diff
@@ -139,28 +133,20 @@ export const LockCell = (item: CellContext<Strategy, number>) => {
   );
 };
 
-export const ActionsCell = (
-  item: CellContext<
-    Strategy,
-    {
-      externalURL?: string | undefined;
-      onClick?: (() => void) | undefined;
-    }
-  >
-) => {
+export const ActionsCell = (_: CellContext<EarnStrategy, unknown>) => {
   const { t } = useTranslation();
 
   return (
     <div className="flex items-center justify-center">
       <Button
-        onClick={item.getValue().onClick}
+        onClick={() => {}}
```

Comment:
why is this an empty function?

Reply:
This cell currently does nothing as we are waiting for Numia to update their services. Meanwhile, I've opened the PR to start the first round of QA